### ISSUE TITLE
Failed to replace `relativeFileDirname` macro in `disasexpl.associations`

### ISSUE THREAD

**Issue by MichaelSuen-thePointer** [dseight/vscode-disasexpl#16]
I wanted to make a source-to-assembly mapping like the following:

`"**/*.c": "${workspaceFolder}/build/${relativeFileDirname}/${fileBasenameNoExtension}.s"`

but when calling `Disassembly Explorer: Show`, it shows error message like this:

`Failed to load file '/home/my_name/my_project/build/${relativeFileDirname}/my_source.s'`

It seems failed to replace such macro

**Comment by danielspaniol** [comment]
This extension uses `relativeFileDir` instead of `relativeFileDirname`.
So the setting
```
"**/*.c": "${workspaceFolder}/build/${relativeFileDir}/${fileBasenameNoExtension}.s"
```
works for me.

Still, I think it would be better if `relativeFileDirname` was supported.

**Comment by dseight** [comment]
Fixed with f6b1829083b134bf224cf80d90a354c0d4fe200d (released in version 0.2.1).
